app/constants/post.ts

```diff
@@ -60,4 +60,5 @@ export default {
         PostTypes.ADD_TO_TEAM,
         PostTypes.REMOVE_FROM_TEAM,
     ],
+    DEFAULT_CHARACTER_LIMIT: 4000,
```

Comment:
I need a confirmation here.  I took this value from v1.  Is it correct ?

Reply:
Seems right. The webapp version has the same value too.